[FIX/ADD] Фикс спрайта для дрона-камеры и добавление сети для нашлемных камер

Guared1365 opened the pull request:
<!-- ЗДЕСЬ должно быть **подробное описание** того, что происходит в PR и зачем это нужно. PR не должен содержать изменений, о которых здесь ничего не сказано. -->
Добавляет канал для нашлемных камер скафандров, который остался на Торче и фиксит отображение спрайта у дрона-камеры журналиста.

<!-- ЗДЕСЬ нужно **привязать ишью**, которые относятся к PR'у в формате `close #1234` или `fixes #1234` - тогда они автоматически закроются вместе с PR. Можно написать `Затрагивает #1234, но не фиксит его`, если вы просто хотите упоминуть ишью, но не закрывать его. -->

<!-- ЗДЕСЬ нужно **расписать изменения** которые попадут в **чейнджлог**, формат - `prefix: краткое описание` -->
### Чейнджлог
```yml
🆑Guared1365
bugfix: Пофикшено отображение спрайта у дрона-камеры журналиста.
tweak: Добавлен канал для нашлемных камер скафандров, который остался на Торче.
/🆑
```

<!--
  Честно заполняем галочки. Чем больше галочек, тем быстрее проверять Pull Request, соответственно он быстрее будет принят.
  Чтобы отметить - ставим `x` (икс) внутри квадратных скобочек вот так: `- [x] ...`.
  Галочки можно доставлять позже по мере окончания работы над PR'ом.
-->

- [x] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашел.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.

Guared1365 committed:
TV Camera sprite fix

Fixes tv camera sprite, so it can display normally.

Guared1365 committed:
Helmet cameras channel adding

Adding helmet cameras channel (network), so everyone can watch it.

LordNest commented on code/game/objects/items/devices/tvcamera.dm:
```diff
@@ -1,6 +1,10 @@
 	desc = "A Ward-Takahashi EyeBuddy livestreaming press camera drone. Weapon of choice for war correspondents and reality show cameramen. It does not appear to have any internal memory storage."
+	// SIERRA-EDIT
+	// Фиксит иконку дроны-камеры, тем самым возвращая её в нормальное состояние.
+	icon = 'icons/obj/tools/photography.dmi'
```
Необходимо добавить иконку из файла `icons/obj/tools/photography.dmi`, соответствующую `icon_state` камеры в файл оверрайда, а не лезть в коркод.

Guared1365 committed:
[FIX] TV Camera sprite fix V2.0

Fixes tv camera sprite, so it can display normally.
Now in override way.

LordNest commented on maps/sierra/sierra_presets.dm:
```diff
@@ -53,6 +53,10 @@ var/global/const/NETWORK_ENGINEERING_OUTPOST = "Engineering Outpost"
 		NETWORK_CALYPSO,
 		NETWORK_PETROV,
 		NETWORK_POD,
+		// SIERRA EDIT
```
Зачем литеррали в файле карты Сьерры "Сьерра-эдит?"

Guared1365 committed:
Helmet cameras channel adding V2.0

Adding helmet cameras channel (network), so everyone can watch it.
Now without annoing comments in code.
